modules/nw-ipfailover-configuration.adoc

```diff
@@ -41,7 +41,45 @@ $ oc adm policy add-scc-to-user privileged -z ipfailover
 ----
 $ oc adm policy add-scc-to-user hostnetwork -z ipfailover
 ----
+
+. {rh-openstack-first} only: Complete the following steps to make the failover VIP addresses reachable on {rh-openstack} ports:
++
+.. Use the {rh-openstack} CLI to find one IP address in the `allowed_address_pairs` parameter of your {rh-openstack} cluster.
++
+[source,terminal]
+----
+$ openstack port show <cluster_name> -c allowed_address_pairs
+----
++
+[source,terminal,subs="attributes,quotes"]
+----
+*Field*                  *Value*
+allowed_address_pairs    ip_address='192.168.0.5', mac_address='fa:16:3e:5d:f0:2d'
+                         ip_address='192.168.0.7', mac_address='fa:16:3e:5d:f0:2d'
+----
 +
+.. Make the failover VIP addresses reachable on {rh-openstack} ports by entering the following command in the {rh-openstack} CLI: 
++
+[source,terminal]
+----
+$ openstack port set <cluster_name> --allowed-address ip-address=192.168.0.5,mac-address=fa:16:3e:5d:f0:2d
+----
++
+.. Create a deployment YAML file to configure IP failover. See "Example deployment YAML for IP failover configuration" in a later step. 
++
+.. Specify the following specification in the deployment so that you set the IP address from the allowed address pair to the `OPENSHIFT_HA_VIRTUAL_IPS` environment variable:
++
+.Example of adding the `192.168.0.5` IP address to `OPENSHIFT_HA_VIRTUAL_IPS`
+[source,terminal,subs="attributes,quotes"]
+----
+# ...
```

Comment:
You can truncate this but you have to give the `apiVersion` and `kind`. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields:~:text=Include%20the%20apiVersion%20and%20kind%20so%20that%20a%20user%20always%20knows%20the%20context%20of%20the%20YAML.

Reply:
:heavy_check_mark: